### PR TITLE
Updated requirements.txt + added note howto install pip-compile

### PR DESCRIPTION
**Pull request by tisnik**
Just and update of requirements.txt used by integration tests. I've also added note howto install pip-compile (it is not obvious for Python apprentices ;)

### CODE REVIEW

**Comment by pkajaba** [comment]
LGTM, Thanks!

File: integration-tests/requirements.txt [hunk 1]
```diff
@@ -4,14 +4,19 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
+# To install pip-compile, run:
+#    pip install pip-tools
+#
```
Comment (jpopelka) [inline integration-tests/requirements.txt]: - since this requirements.txt is generated (by the pip-compile) your note will most likely be lost with some future update
- we use python3 and pip3, so the correct command is `pip3 install pip-tools`

File: integration-tests/requirements.txt [hunk 2]
```diff
@@ -4,14 +4,19 @@
...
 behave==1.2.5
 docker-py==1.10.6
 docker-pycreds==0.2.1     # via docker-py
+enum34==1.1.6             # via parse-type
```
Comment (jpopelka) [inline integration-tests/requirements.txt]: This was added because you used python2 version of pip-compile. If you installed pip-compile with `pip3 install pip-tools` this would be not added, because `enum34` is a back-port of python3.4's enum to python2.

File: integration-tests/requirements.txt [hunk 3]
```diff
@@ -4,14 +4,19 @@
...
 jsonschema==2.5.1
 parse-type==0.3.4         # via behave
 parse==1.6.6              # via behave, parse-type
 py==1.4.31                # via pytest
 pytest==3.0.3
+repoze.lru==0.6           # via jsonschema
```
Comment (jpopelka) [inline integration-tests/requirements.txt]: This seems to be the same case as with the above `enum34`.

Comment (tisnik) [inline integration-tests/requirements.txt]: You are right, thanks. Do you want another PR?

Comment (jpopelka) [inline integration-tests/requirements.txt]: Yes please. You can create one easily by pressing `Revert`button next to the *pkajaba merged commit c1586c4 into fabric8-analytics:master* message.
